keepalived/check/check_http.c

```diff
@@ -123,6 +123,38 @@ alloc_http_get(char *proto)
 	return http_get_chk;
 }
 
+static int
```

Comment:
I prefer type bool, return true for same (currently 0), and false if different and change goto err to return false